packages/components/src/components/relative-datepicker/relative-datepicker.tsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { toMoment, daysFromTodayTo } from '@deriv/shared';
 import Text from '../text';
 
-const RelativeDatepicker = ({ onChange, min_date, max_date, title }) => {
-    const hidden_input_ref = React.useRef();
+type TRelativeDatepickerProps = {
+    max_date?: number | null;
```

Comment:
Should max_date & min_date have null as type? Since they are optional, I think it can be ignored

Reply:
@likhith-deriv, yes. if you look at the file  `duration-mobile.jsx`, the function called there returns and array of `[number | null]`. This is why I added `null` to the definition

Reply:
<img width="1059" alt="Screenshot 2022-10-31 at 2 38 59 PM" src="https://user-images.githubusercontent.com/104334373/198946886-30f2d92f-b3bd-456b-ac28-b8c071b4cf20.png">